usbtmc/usbtmc.py

```diff
@@ -696,51 +718,77 @@ def clear(self):
             raise UsbtmcException("Clear failed", 'clear')
 
     def remote(self):
-        "Send remote command"
+        """
+        Send the remote command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def local(self):
-        "Send local command"
+        """
+        Send the local command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def lock(self):
-        "Send lock command"
-
+        """
+        Advantest/ADCMT ONLY: Send the lock command to the instrument.
```

Comment:
This is for a quirk that was submitted in a PR.  Perhaps moving them into dedicated methods would be better.  A subclass is not necessary unless you want to turn this class into a factory.